libtransact/src/state/merkle/sql/mod.rs

```diff
@@ -15,7 +15,499 @@
  * -----------------------------------------------------------------------------
  */
 
-mod migration;
+//! SQL-backed merkle-radix state implementation.
+
+pub mod backend;
```

Comment:
Should this actually be non-pub? Below, there is "pub use" of Backend/Connection; feels like there should only be one approach.

Reply:
I've changed the module to be pub, and removed the `pub use`